.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: weekly
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: weekly
```

Some cosmetic suggestions:

```suggestion
      interval: weekly
      day: 'monday'
      time: '00:00'
  commit-message:
      prefix: 'chore(deps):'
```

---

@fbidu 👍